internal/metrics/system.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import (
+	"github.com/jakub-dzon/k4e-device-worker/internal/configuration"
+	"github.com/jakub-dzon/k4e-operator/models"
+	"reflect"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	systemTargetName                     = "system"
+	DefaultSystemMetricsScrapingInterval = 60
```

Comment:
we have a different interval for workloads and system? can u explain the rational?

Reply:
we have a different interval for workloads and system? can u explain the rational?

Reply:
Users might have different purpose metrics generated by their workloads and system metrics might be more or less important for them or be more or less variable, hence the possibility to set separate, custom interval. 

Default is the same for workload and system metrics and for convenience it is defined separately - to avoid accidental change to one when willing to modify only another.

Reply:
the same can be said about specific metrics within each group. E.g. IP address metric is different from bytes-used metric. But both are system. But it doesn't hurt to have a separate interval.